lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/dva/DvaCryptographyService.java

```diff
@@ -101,10 +101,22 @@ private JWSObject createJWS(String stringToSign) throws JOSEException, JsonProce
         return jwsObject;
     }
 
-    private JWEObject createJWE(String data) throws JOSEException, CertificateException {
+    private JWEObject createJWE(String data)
```

Comment:
Were missing a test for this file can we add one now it differs from the DCS equivalent